regression_tests/actor_service.go

```diff
@@ -0,0 +1,111 @@
+package regressiontests
+
+import (
+	"context"
+	"fmt"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/commands"
+	"github.com/bitcoin-sv/spv-wallet-go-client/config"
+	"github.com/bitcoin-sv/spv-wallet-go-client/walletkeys"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+)
+
+// Actor represents an individual user or entity within the SPV Wallet ecosystem.
+// It includes details like the alias, private key (xPriv), public key (xPub), and paymail address.
+type Actor struct {
```

Comment:
All the types could be not-exported. 
I think it would be beneficial to not confuse others that these structs are used somewhere else.